docs/backup_and_restore.rst

```diff
@@ -104,6 +104,27 @@ archive in the output of the backup command.
 
 .. include:: includes/backup-warning.txt
 
+.. _backing_up_logos:
+
+Backing up your custom logo
+'''''''''''''''''''''''''''
+
+As of SecureDrop |version|, the backup will not include any custom logo file
+that you may have configured for your SecureDrop instance. To back up the logo,
+download it using Tor Browser or on the command line. From ``~/Persistent``
+(a level above the ``securedrop`` directory), the following command will
```

Comment:
It might be better to tell folks to open a terminal and explicitly include the command, ie:

Open a terminal via **Applications > System Tools > Terminal** and run the following commands:

```
cd ~/Persistent
wget ......
````
Also, if you want to remove the v2/v3 stuff, you could grab the canonical source address from the shortcut file! (just realized this!) with `cat ~/Desktop/source.desktop | grep Exec | awk '{print $2}'`

Reply:
Nice! Simplified a bit further and used in beef2f0da18fa6276631112d862d4afe925a817f